printer_mods/Phalanx/Other-V2-Idlers/README.md

```diff
@@ -0,0 +1,55 @@
+# "Other" V2 Idlers
+ Alternative Idlers that allow the usage of "Full-Figured" toolheads without the 10mm loss of travel at the front corners of the bed.
+
+These idlers were developed with the goal to allow the usage of wider toolheads with the Voron V2, better align the tensioning screw to the centerline of the idler shaft, add guides to the parts to allow them to interlock and provide a robust tensioning system, all while streamlining the process of making fine adjustments to tensioning. 
+This design iteration has been running on my machine for 50+ hrs with no issues or print quality degradation.  
+
+The idler design is ambidextrous to allow simplified batch printing. All parts are converted to stl in optimal printing orientation with quantities specified per Voron standards: 
+
+![image](https://user-images.githubusercontent.com/65414930/123886695-a1619680-d915-11eb-8a33-212a3cb1b8ef.png)
+
+
+# Print Settings:
+All parts should be printed to standard Voron print settings:
+
+![image](https://user-images.githubusercontent.com/65414930/123886729-b63e2a00-d915-11eb-81b5-52c1082b56c9.png)
+
+Note: If your printer is not properly tuned i.e. printing dimensionally-accurate parts free from functional defects, the slide mechanism WILL NOT work proerly. 
+
+# BOM:
+2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source. 
+4 ea. F695 bearings
+4 ea. 0.5mm thick 5mm shims
+2 ea. M3x35mm length SHCS - Stock 40mm may be run if not using the bolted door hinge mod. 
```

Comment:
Add a note here that the screw must be fully threaded, as some long screws linked from the sourcing guide have a shoulder at the top and are not fully threaded.